test/y2partitioner/dialogs/partition_resize_test.rb

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "../test_helper"
+
+require "yast"
+require "cwm/rspec"
+require "y2storage"
+require "y2partitioner/dialogs/partition_resize"
+
+describe Y2Partitioner::Dialogs::PartitionResize do
+  using Y2Storage::Refinements::SizeCasts
+
+  before do
+    devicegraph_stub("mixed_disks.yml")
+  end
+
+  let(:current_graph) { Y2Partitioner::DeviceGraphs.instance.current }
+
+  # Creates a new partition
+  let(:partition) do
+    sdc = Y2Storage::Disk.find_by_name(current_graph, "/dev/sdc")
+    sdc.partition_table.create_partition("/dev/sdc1", Y2Storage::Region.create(2048, 1048576, 512),
+      Y2Storage::PartitionType::PRIMARY)
+  end
+
+  let(:resize_info) do
+    instance_double(Y2Storage::ResizeInfo,
+      resize_ok?: true,
+      min_size:   10.MiB,
+      max_size:   100.GiB)
+  end
+
+  subject { described_class.new(partition, resize_info) }
+
+  include_examples "CWM::Dialog"
+
+  describe "#contents" do
+    def find_label(contents, text)
+      contents.nested_find do |widget|
+        widget.respond_to?(:value) &&
+          widget.value == :Label &&
+          widget.params.any? { |i| i.include?(text) }
+      end
+    end
+
+    it "contents a widget for selecting the new size" do
```

Comment:
I guess you mean "it contains a widget". Content as a verb means to satisfy someone/something, so "it contents a widget" would mean it makes the widget happy.